internal/controller/controllers/installenv_controller_test.go

```diff
@@ -146,6 +147,86 @@ var _ = Describe("installEnv reconcile", func() {
 		Expect(conditionsv1.FindStatusCondition(installEnvImage.Status.Conditions, adiiov1alpha1.ImageCreatedCondition).Status).To(Equal(corev1.ConditionFalse))
 	})
 
+	It("create new installEnv image - cluster not retrieved from database", func() {
+		clusterDeployment := newClusterDeployment("clusterDeployment", testNamespace, getDefaultClusterDeploymentSpec("clusterDeployment-test", "pull-secret"))
```

Comment:
if you have a lot of common code in the test you probably should create a single function for it or put it under a single context and create common resources with `BeforeEach`